content/cli/v6/using-npm/disputes.md

```diff
@@ -12,7 +12,7 @@ disputes with other npm publishers. It also describes special steps you should
 take about names you think infringe your trademarks.
 
 This document is a clarification of the acceptable behavior outlined in the
-[npm Code of Conduct](https://www.npmjs.com/policies/conduct), and nothing in
+[npm Code of Conduct](https://docs.npmjs.com/policies/conduct), and nothing in
```

Comment:
This needs to be updated in https://github.com/npm/cli/blob/v6/docs/content/using-npm/disputes.md